### ISSUE TITLE
When installing the flatpak on Steam Deck, Discover can't view installed apps anymore without crashing

### ISSUE THREAD

**Issue by NathanDreamweaver** [Brod8362/d4m#25]
As per title. Additionally, it also complains about a missing/not found repo whenever launching Discover, related to d4m.

**Comment by Brod8362** [comment]
Thanks for the issue. I've been extremely busy lately and haven't had much time to devote to d4m. 

I don't know what "Discover" is. I don't have a steam deck so I've never actually been able to test the software myself. Can you explain what "Discover" is with some context that might be helpful? 

Thanks!

**Comment by NathanDreamweaver** [comment]
Discover is just the KDE software center! It lets you install stuff via a GUI.

**Comment by Brod8362** [comment]
Aah okay, thanks. If I can find the time this weekend I'll set up a VM with KDE and see if I can replicate and fix this behavior.

Actually, I've had 0.3.3 (next major version) on the chopping block for a while, so I'm going to try and finally release that this weekend. It may already be fixed and we just don't know it yet.

**Comment by NathanDreamweaver** [comment]
Thanks! The flatpak is the only method to run d4m on the deck, as far as I can tell, since system is read only. Can't install libarchive or setup python.

**Comment by Brod8362** [comment]
Yeah that's unfortunately the case, and actually the only reason I bother with flatpak to begin with. 
If you want to contact me on discord I can try sending you some beta flatpaks and see if it fixes the problem enough for you to use the software. Handle Brod8362#2928

**Comment by youjackal** [comment]
I am having the exact same issue too!!
